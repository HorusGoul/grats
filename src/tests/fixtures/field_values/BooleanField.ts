/** @GQLType */
export default class Query {
  /** @GQLField */
  haveBeenGreeted(): boolean {
    return false;
  }
}
