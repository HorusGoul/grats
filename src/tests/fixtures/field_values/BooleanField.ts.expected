-----------------
INPUT
----------------- 
/** @GQLType */
export default class Query {
  /** @GQLField */
  haveBeenGreeted(): boolean {
    return false;
  }
}

-----------------
OUTPUT
-----------------
directive @renameField(name: String!) on FIELD_DEFINITION

type Query {
  haveBeenGreeted: Boolean
}